game/scripts/vscripts/items/siege_mode.lua

```diff
@@ -58,6 +73,31 @@ end
 
 --------------------------------------------------------------------------------
 
+function modifier_item_siege_mode_siege:OnCreated( event )
+	local spell = self:GetAbility()
+
+	spell.mod = self
+
+	self.atkRange = spell:GetSpecialValueFor( "siege_attack_range" )
```

Comment:
Same deal, needs refresh handling. you can also remove the or's if you make this change